File: app/src/main/java/com/stevenfrew/beatprompter/cloud/CloudFolderSearchListener.java
package com.stevenfrew.beatprompter.cloud;

public interface CloudFolderSearchListener extends CloudListener {
    void onCloudItemFound(CloudItemInfo cloudItem);
    void onFolderSearchError(Throwable t);
    void onFolderSearchComplete();
    void onProgressMessageReceived(String message);
}

File: app/src/main/java/com/stevenfrew/beatprompter/cloud/CloudFolderSearchListener.kt
package com.stevenfrew.beatprompter.cloud

interface CloudFolderSearchListener : CloudListener {
    fun onCloudItemFound(cloudItem: CloudItemInfo)
    fun onFolderSearchError(t: Throwable)
    fun onFolderSearchComplete()
    fun onProgressMessageReceived(message: String)
}
